data/net/minecraft/world/phys/shapes/DiscreteCubeMerger.mapping

```diff
@@ -1,3 +1,6 @@
 CLASS net/minecraft/world/phys/shapes/DiscreteCubeMerger
+	METHOD <init> (II)V
+		ARG 1 aa
+		ARG 2 bb
```

Comment:
What are these?

Reply:
So from my understanding looking at the greater context of their use.
AA and BB in this context are two lists of two shapes split into lists of coordinates of the shapes in X-Axis orientation.
Basically they hold a set of double-sized coordinates of some part of the shapes as two lists.

The best name I could come up with is AA and BB since in other parts of the code inputs to this is mapped as AA and BB.
If someone can come up with a better name later on, that would be great :P 
If it's prefered I can remove the AA BB mappings and wait for someone else to contribute a better interpretation.

Reply:
If there's other context which gives these mappings meaning then that's fine for me, I just didn't see any in the immediate vicinity.

Reply:
I've asked @OrionDevelopment to do a review over the shape related stuff and provide feedback, including on this